src/components/common/Banner.js

```diff
@@ -28,6 +28,8 @@ function Banner() {
     return bannerContent('Apply to Course', 'Select Courses');
   } else if (path === '/instructor-add-course') {
```

Comment:
Instead of using if statements, have you considered using a case statement?

Reply:
if thats how we feel we should go I wouldn't mind. I just followed the format